CodeYourFuture/cyf-forms
# Feature/697-move-employer-names-to-backend-code-rever-all-changes-merged-to-qa-branch[QA]

This is a:


**Purpose**
This PR aims to revert all the changes merged to the QA branch. In other words, bringing back where the QA branch was before this implementation.


**How to check** 

1. Visit this branch: **feature/697-move-employer-names-to-backend-code**.
2. Run cyf-api repo.
3. Run your cyf-forms repo, when the form loads you should be able to see all the employers' names. The employers' list is coming from the front-end code.



**Links**
issue: https://github.com/CodeYourFuture/tech-team/issues/697
cyf-api:  https://github.com/CodeYourFuture/cyf-api/pull/635



### Author checklist

- [x] I have made the PR to `qa` from a branch named **feature/697-move-employer-names-to-backend-code**.
- [x] I have completed the manual tests [described here](https://github.com/CodeYourFuture/tech-team/wiki/3.-How-we-work#manual-test-procedures)
